src/modules/integrator-guide/pages/security-model/permissions-evaluation.adoc

```diff
@@ -0,0 +1,315 @@
+----
+{
+  "image": {
+    "rel": "security-model/access-control/permission-button-page.png",
+    "alias": "new-user",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "annotations": [{
+    "kind": "box",
+    "padding": "xs",
+    "x": 1535,
+    "y": 75,
+    "w": 130,
+    "h": 42
+  }]
+}
+----
+
+2.Click the *[add +]* button.
+An evaluation window appears with two dropdown options, the *[select role(s)]* and *[search or select user]*.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "security-model/access-control/add-role.png",
+    "alias": "add-role",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "annotations": [{
+    "kind": "box",
+    "padding": "xs",
+    "x": 765,
+    "y": 150,
+    "w": 750,
+    "h": 80
+}]
+}
+----
+
+3.Click on the *[select roles]* or the *[select user]* dropdown and select a role or a user to evaluate.
+Only one of two options can be applied.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "security-model/access-control/eval-permission.png",
+    "alias": "eval-permission.png",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "focus": {
+    "padding": "xs",
+    "x": 604,
+    "y": 320,
+    "w": 1232,
+    "h": 684
+  },
+  "annotations": [{
+    "kind": "box",
+    "x": 600,
+    "y": 330,
+    "w": 735,
+    "h": 270
+  }]
+}
+----
+
+Click on the btn:[save] button.
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "security-model/access-control/save-evaluation-permission.png",
+    "alias": "save-evaluation-permission",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "focus": {
+    "padding": "xs",
+    "x": 344,
+    "y": 42,
+    "w": 1232,
+    "h": 630
+  },
+  "annotations": [{
+    "kind": "box",
+    "x": 1150,
+    "y": 613,
+    "w": 193,
+    "h": 38
+  }]
+}
+----
+
+4.If you chose role evaluation in step 3, repeat the step again, choosing a different role in order to evaluate the permissions applied to the different roles.
+
+.The screenshot shows an overview of permissions appied on the roles being evaluated.
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "security-model/access-control/save-eval.png",
+    "alias": "save-eval",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "annotations": []
+}
+----
+
+== Permission Evaluation on Resource Components 
+
+Corteza has three major resource components, namely;
+*System*, *compose* and *automation* components.
+Each resource component has operations that are performed in it.
+
+[IMPORTANT]
+====
+Permissions to the various operations within a component resource are subject to the roles assigned to a user.
+====
+
+Permissions can be evaluated by selecting a combination of roles or a particular user.
+
+Follow these steps to access and evaluate the system, compose and automation permissions.
+
+1.Navigate to your {PRODUCT_NAME} instance *{LOCAL_HOSTNAME}* and click the *[Admin area]* application.
```

Comment:
> Navigate to your {PRODUCT_NAME} instance *{LOCAL_HOSTNAME}* and click the...

Indicates that your instance is on that address which is not always true.
Make it know that it is an example link.

I'd do something like 

> Navigate to your {PRODUCT_NAME} instance (such as *{LOCAL_HOSTNAME}*) and click the...